WelcomeHome/WelcomeHome.DAL/Models/EventType.cs

```diff
@@ -1,11 +1,16 @@
 ﻿using System.ComponentModel.DataAnnotations;
 
-namespace WelcomeHomeModels.Models
+namespace WelcomeHome.DAL.Models
 {
     public class EventType
     {
         [Key]
         public int Id { get; set; }
         public string Name { get; set; } = null!;
+        public List<Event> Events { get; set; }
```

Comment:
ICollection<Event> here
https://stackoverflow.com/questions/7655845/icollectiont-vs-listt-in-entity-framework